### DISCUSSION TITLE
使用Vercel搭建，很容易被gfw给墙了，怎么办？

### DISCUSSION THREAD

**Discussion by henryhu712** [ChatGPTNextWeb/NextChat#2800] [Q&A]
我的域名已经被墙了两个。有什么好的办法来避免被墙？

**Comment by iamsophie** [comment]
同问？

这个是不是应该可以改造 一下，把登录页独立出来，没有登录的先跳到登录页，这样被墙的概率是不是低了一些？

**Reply by Flyingpen** [reply]
这是好主意，我也被墙了，现在域名跳转反炸中心，/(ㄒoㄒ)/~~

**Reply by Physton** [reply]
可以看下我修改编译的docker容器：https://hub.docker.com/r/physton/chatgpt-next-web

- 默认发送键换成 Enter
- 默认模型换成 gpt-3.5-turbo-16k
- 默认全屏模式
- ChatGPT Next 标题/文字换成 Chat AI
- 设置了授权码 code 的情况下，默认进入授权页面，而不是进入聊天页面。并且必须是授权成功的状态，才会显示聊天页面。

**Comment by Yidadaa** [comment]
套上国内的 cdn 全站加速即可解决此问题，不过如果是域名污染导致的墙，那就只能加登录鉴权。
